src/coreclr/vm/dllimportcallback.h

```diff
@@ -148,6 +148,9 @@ class UMEntryThunkData
     // if m_pObjectHandle is non-NULL, this field is still set to help with diagnostic of call on collected delegate crashes
     // but it may not have the correct value.
     PCODE                   m_pManagedTarget;
+    // The native code to tailcall. May or may not match what the UM entry precode has been patched to call directly.
+    // This is populated by RunTimeInit at the end of execution.
+    PCODE                   m_pCachedCallTarget;
```

Comment:
```suggestion
#ifdef FEATURE_INTERPRETER
    // InterpreterPrecode to tailcall if the target is interpreted. This allows TheUMEntryPrestubWorker 
    // stash the hidden argument in a thread static and avoid collision with the hidden argument 
    // used by InterpreterPrecode.
    Volatile<PCODE>         m_pIntepretedCallTarget;
#endif
```
The new field should be interpreter only.